manifests/volume.pp

```diff
@@ -163,7 +163,7 @@
       # this volume exists
 
       # our fact lists bricks comma-separated, but we need an array
-      $vol_bricks = split( getvar( "::gluster_volume_${title}_bricks" ), ',')
+      $vol_bricks = getvar( "::gluster_volume_${title}_bricks" )
       if $bricks != $vol_bricks {
```

Comment:
@coder-hugo Do you think the arrays can be sorted before they're compared (and operated on)?

This would fix https://github.com/voxpupuli/puppet-gluster/issues/121 ?